### DISCUSSION TITLE
Prometheus - Disable exemplars globally

### DISCUSSION THREAD

**Discussion by vishiy** [grafana/grafana#41903] [Observability]
Looking for an easier way to disable exemplars globally for our Prometheus dashboard queries, rather than doing it per query panel. Is it possible ? Also is it possible to disable thru grafonnet API ?

**Comment by shaunmansell** [comment]
I'm also looking for a way to turn off exemplar queries globally for Prometheus datasources that don't support them. 

The fact that any new query defaults exemplars on irrespective of whether the call will return anything is resulting in a lot of extra network calls on larger dashboards that don't need to happen.

We code commit dashboard JSON files to deploy out to Grafana instances so we're dealing with this through updating repo content directly, but it'd be much nicer if these calls could be toggled off in Grafana against the datasource.

**Comment by ivanahuckova** [comment]
In https://github.com/grafana/grafana/pull/45260 that is part of 8.5.0-beta1 release we have change the default value for exemplars to false. I know that this is not exactly what you look for in this discussion, but I thought that it might partly fix your issue.

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). Due to the age and number of responses to this discussion, we are deciding to close it. If this is something you would like to see in Grafana, feel free to open an [issue](https://github.com/grafana/grafana/issues) so the discussion can continue. Thank you!
